ui/src/app/settings/components/project-details/resource-lists-panel.tsx

```diff
@@ -183,14 +183,14 @@ function viewDestinationServiceAccountsInfoList(type: field, proj: Project) {
                 <React.Fragment>
                     <div className='row white-box__details-row'>
                         <div className='columns small-4'>Server</div>
-                        <div className='columns small-8'>Namespace</div>
-                        <div className='columns small-12'>DefaultServiceAccount</div>
+                        <div className='columns small-3'>Namespace</div>
```

Comment:
Why is this change required ?

Reply:
I agree, it looks better with your change. I have approved your PR. Thanks for working on it.